sources/packages/api/src/route-controllers/institution-type/institution-type.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller, Get } from "@nestjs/common";
+import BaseController from "../BaseController";
+import { InstitutionTypeService } from "../../services";
+import { OptionItem } from "../../types";
+@Controller("institution/type")
+export class InstitutionTypeController extends BaseController {
+  constructor(private readonly institutionTypeService: InstitutionTypeService) {
+    super();
+  }
+
+  @Get("options-list")
+  async getOptionsList(): Promise<OptionItem[]> {
+    const institutionTypes = await this.institutionTypeService.getAll();
+    return institutionTypes.map((t) => ({
```

Comment:
instead of `t` can you give some meaningful name

Reply:
Thanks for noticing @ann-aot . It is changed now.